src/Service/DashboardHelper.php

```diff
@@ -12,186 +12,184 @@
 
 class DashboardHelper
 {
-    private BoardRepository $boardRepository;
-    private CaseEntityRepository $caseRepository;
-    private TranslatorInterface $translator;
-    private UrlGeneratorInterface $router;
-    private UserRepository $userRepository;
-
-    public function __construct(BoardRepository $boardRepository, CaseEntityRepository $caseRepository, TranslatorInterface $translator, UrlGeneratorInterface $router, UserRepository $userRepository)
+    public function __construct(private BoardRepository $boardRepository, private CaseEntityRepository $caseRepository, private TranslatorInterface $translator, private UrlGeneratorInterface $router, private UserRepository $userRepository)
     {
-        $this->boardRepository = $boardRepository;
-        $this->caseRepository = $caseRepository;
-        $this->translator = $translator;
-        $this->router = $router;
-        $this->userRepository = $userRepository;
     }
 
     public function getDashboardGridInformation(Municipality $municipality, User $user): array
     {
         $gridInformation = [];
 
-        $gridInformation = $this->addUserColumnCaseInformation($user, $gridInformation);
+        $gridInformation[] = $this->getUserColumnCaseInformation($user);
 
-        $gridInformation = $this->addBoardsColumnCaseInformation($municipality, $gridInformation);
+        $gridInformation = array_merge($gridInformation, $this->getBoardsColumnCaseInformation($municipality));
```

Comment:
This looks inconsistent. It should read something like

```php
$gridInformation[] = $this->getUserColumnCaseInformation($user);
$gridInformation[] = $this->getBoardsColumnCaseInformation($municipality);
```

Reply:
Renamed `getBoardsColumnCaseInformation` to `getBoardColumnsCaseInformation` to make it clearer that it handles all boards in provided municipality.